decidim-conferences/app/controllers/decidim/conferences/conferences_controller.rb

```diff
@@ -54,6 +54,18 @@ def promoted_conferences
       def stats
         @stats ||= ConferenceStatsPresenter.new(conference: current_participatory_space)
       end
+
+      def conference_participatory_processes
+        @conference_participatory_processes ||= @current_participatory_space.linked_participatory_space_resources(:participatory_processes, "included_participatory_processes")
+      end
+
+      def conference_assemblies
+        @conference_assemblies ||= @current_participatory_space.linked_participatory_space_resources(:assemblies, "included_assemblies")
+      end
+
+      def conference_consultations
+        @conference_consultations ||= @current_participatory_space.linked_participatory_space_resources("Consultation", "included_consultations")
```

Comment:
Why are you using a String here instead of a Symbol?

Reply:
because if I use a symbol `:consultations` it returns  a `Question`